workshops/templates/workshops/eventrequest.html

```diff
@@ -0,0 +1,22 @@
+{% extends "workshops/_page.html" %}
+
+{% block content %}
+{% include "workshops/eventrequest_email_html.html" %}
+
+<div class="clearfix">
+  <p class="edit-object pull-left">
+    {% if perms.workshops.change_eventrequest and perms.workshops.add_event %}
+    <a href="{% url 'eventrequest_accept' object.pk %}" class="btn btn-success">Accept</a>
+    {% else %}
+    <a href="#" class="btn btn-success disabled">Accept</a>
```

Comment:
The links aren't clickable, so there's no point in having a full address in that `href`.